doc/source/walkthrough.rst

```diff
@@ -1,14 +1,24 @@
-Walkthrough
-===========
+Ray Core Walkthrough
```

Comment:
Do we want to start calling the main API `Ray Core`? Not sure about this, it makes me wonder what `Ray` is if this is `Ray Core`. Also, from the library perspective it furthers the image that tune and rllib are "part" of ray and not libraries written on top of it

Reply:
It's very likely that someone click `Ray Walkthrough` expecting content about RLlib or Tune. The goal here is to highlight this as core primitives.

Reply:
I think over time we'll want to distinguish Ray from the many libraries that exist on top of it, where RLlib, Tune, etc. are the first ones. Then just saying "Ray" to refer to the core library would be sufficient, IMHO.